### PR TITLE
Add osuidol announcement

### PR DESCRIPTION
**Pull request by ModuleLFS**
Description
------
osu!idol 2017 announcement

Status
------
Finished, waiting for approval and merge

### CODE REVIEW

File: news/2017-08-11-osu-idol-2017-auditions-start.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+layout: post
+title: "osu!idol 2017 Auditions start!"
+date: 2017-08-11 08:30:00 +0000
+---
+
+Do you like singing? How about taking part in a singing competition? Show your talent to the osu!world and become this year's osu!idol!
+
+![](/wiki/shared/news/2017-08-11-osu-idol-2017-auditions-start/banner.jpg)
+
+WHAT IS THIS ALL ABOUT?
```
Comment (sr229) [inline news/2017-08-11-osu-idol-2017-auditions-start.md]: L11: use Header L3 here

File: news/2017-08-11-osu-idol-2017-auditions-start.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+-----
+
+osu!idol is where contestants as a solo or a duet sing their way through multiple stages of different genres with the aim to reach the very top of the idol podium! Our judges will ensure the best is standing at the final stretch and you - the community, will decide with a voting to determine our next osu!idol! So get hyped, grab your microphones and get ready to sing your lungs out! 
+
+BUT WAIT, ARE THERE ANY PRIZES?
```
Comment (sr229) [inline news/2017-08-11-osu-idol-2017-auditions-start.md]: L16: Use header L3 here

File: news/2017-08-11-osu-idol-2017-auditions-start.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+-----
+
+* **1st:** osu!idol 2017 Winner profile Badge + 6 months supporter tag.
+* **2nd:** 3 months supporter tag.
+* **3rd:** 2 months supporter tag. 
+
+SO WHERE DO I SIGN UP?
```
Comment (sr229) [inline news/2017-08-11-osu-idol-2017-auditions-start.md]: L23: use header L3 here

**Comment by ModuleLFS** [comment]
Announcement is following previous templates and I have no clue what is an L3 header.

**Comment by ModuleLFS** [comment]
I have used the latest announcement as template as it had all cap headers.

**Pull request merged**